oneapp-server/types/Application.js

```diff
@@ -31,26 +31,32 @@ const resolvers = {
     }),
   },
   Mutation: {
-    applicationUpdate: async (_parent, { input }, { auth }) => {
+    applicationUpdate: async (_parent, { input }, { dataSources, auth }) => {
       const APPLICATION_NUMBER = auth.user.USER_ID;
       logger.info('Performing upsert on application (%s)', APPLICATION_NUMBER);
 
       const application = {
         APPLICATION_NUMBER,
       };
```

Comment:
I'm good with remove the application information from the mutation, as you pointed out in stand-up today it would make sense for the front-end to call the query again after running the mutation.